spec/lib/services/obm-api-service-spec.js

```diff
@@ -6,7 +6,6 @@
 describe("Http.Services.Api.Obms", function () {
     var obmService;
     var Promise;
-    var chai = require('chai');
     chai.use(require('chai-string'));
```

Comment:
Remove this line is just to fix the unit-test error, the unit-test will fail with following error information if keep this line:
```
  1) Http.Services.Api.Obms getObmLib Run getObm:
     TypeError: Object [object Object] has no method 'equalIgnoreCase'
      at Context.<anonymous> (spec/lib/services/obm-api-service-spec.js:205:40)

  2) Http.Services.Api.Obms getObmLibById Run getObmLibById:
     TypeError: Object [object Object] has no method 'equalIgnoreCase'
      at Context.<anonymous> (spec/lib/services/obm-api-service-spec.js:225:40)
```

If remove this line, the error disappears. I don't understand why at present. But anyway, I  think the best is to move the [chai-string](http://chaijs.com/plugins/chai-string/) to on-core/spec/helper.js, As I see the chai-string is good for string assertion and potentially will be used by other repos.